tachyon/crypto/commitments/vector_commitment_scheme.h

```diff
@@ -74,6 +74,25 @@ class VectorCommitmentScheme {
     const Derived* derived = static_cast<const Derived*>(this);
     return derived->DoCommit(container, random_value, result);
   }
+
+  // Create a opening proof that proves that |members| belong to a
+  // commitment.
+  template <typename ContainerTy, typename Proof>
+  [[nodiscard]] bool CreateMultiOpeningProof(const ContainerTy& members,
```

Comment:
Is there any reason for the name `CreateMultiOpeningProof`? Since it may be a open in a single point, `CreateProof` or `CreateOpeningProof` seems more appropriate.
```suggestion
  [[nodiscard]] bool CreateProof(const ContainerTy& members,
```

Reply:
I'll go with `CreateOpeningProof`!